src/compute/src/logging/reachability.rs

```diff
@@ -77,36 +81,44 @@ pub(super) fn construct<A: Allocate>(
         use timely::dataflow::channels::pact::Pipeline;
         let mut input = flatten.new_input(&logs, Pipeline);
 
-        let (mut updates_out, updates) = flatten.new_output::<ConsolidatingContainerBuilder<_>>();
+        type RUpdates = <(
```

Comment:
Maybe a comment here about what is going on? `RUpdates` isn't a clear enough name for me to unpack all the tuple components.

Reply:
Renamed to `UpdatesRegion`.